test/e2e/data/infrastructure-docker/v1alpha4/cluster-template-ipv6/kcp-ipv6.yaml

```diff
@@ -12,7 +12,7 @@ spec:
     initConfiguration:
       localAPIEndpoint:
         advertiseAddress: '::'
-        bindPort: 6443
+        bindPort: 7777
```

Comment:
I would prefer not changing this port in the template for v1alpha4.

Afaik the v1alpha4 templates are only used for clusterctl upgrade test and those tests should not be running with IPv6, so I think it's not necessary. (also not for the non-IPv6 KCP)